components/layouts/DocsLayout.js

```diff
@@ -41,6 +41,9 @@ export default function DocsLayout({ children, title }) {
           <div className="flex flex-grow flex-row">
             <MDXProvider components={ComponentStyle}>
               <div className="w-full max-w-7xl px-4 sm:px-6 lg:px-8 dark:text-white">
+                <h3 className="mb-8 font-bold text-xl md:text-3xl">
+                  You can setup LinkFree code base in the following two ways.
+                </h3>
```

Comment:
Does this need to be on every page?

Reply:
no actually we do not need it for all. I'll change and push it